source/fundamentals/auth.txt

```diff
@@ -0,0 +1,384 @@
+=========================
+Authentication Mechanisms
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
```

Comment:
Given the fact that you list each of the auth mechanisms below, I'm not sure if we need the "on this page" widget here. It feels repetitive to me.

Reply:
I think it's important for a lengthy page like this one to have a ToC at the top since we establish this as an expected pattern in our docs. I made the decision to add links into the bulleted list (not present in the Node driver docs) because the examples are much longer and don't fit in the typical viewport. If you strongly feel this is repetitive, I can remove the links from the bulleted list as it was a late refinement.